recipes/argopt/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "argopt" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: c8ca9cdeb6bf5e8d6b0fb6a6cd3c676210fd3ebacdabab57f5b95d1ed514f0af
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - argopt
+
+about:
+  home: "https://github.com/casperdcl/argopt"
+  license: "MPL-2.0"
+  license_family: "MOZILLA"
```

Comment:
You still want to include the line:

`license_file: LICENSE`

Reply:
Added it back. Thanks!

Reply:
oops, there is a spelling error in the source distribution. It should be `license_file: LICENCE`

Reply:
Thanks for looking over it carefully. I definitely missed that.